test/mocha-utils.js

```diff
@@ -0,0 +1,56 @@
+const {TestServer} = require('../utils/testserver/index');
+const path = require('path');
+const puppeteer = require('../');
+
+const setupServer = async() => {
+  const assetsPath = path.join(__dirname, 'assets');
+  const cachedPath = path.join(__dirname, 'assets', 'cached');
+
+  const port = 8907;
+  const server = await TestServer.create(assetsPath, port);
+  server.enableHTTPCache(cachedPath);
+  server.PORT = port;
+  server.PREFIX = `http://localhost:${port}`;
+  server.CROSS_PROCESS_PREFIX = `http://127.0.0.1:${port}`;
+  server.EMPTY_PAGE = `http://localhost:${port}/empty.html`;
+  // const httpsPort = port + 1;
+  // httpsServer = await TestServer.createHTTPS(assetsPath, httpsPort);
+  // httpsServer.enableHTTPCache(cachedPath);
+  // httpsServer.PORT = httpsPort;
+  // httpsServer.PREFIX = `https://localhost:${httpsPort}`;
+  // httpsServer.CROSS_PROCESS_PREFIX = `https://127.0.0.1:${httpsPort}`;
+  // httpsServer.EMPTY_PAGE = `https://localhost:${httpsPort}/empty.html`;
```

Comment:
this stuff will be enabled again - just haven't yet migrated a test that needs it.

This setup is lifted right from our custom test runner.